src/core/audio/AudioFileFlac.cpp

```diff
@@ -94,7 +98,10 @@ void AudioFileFlac::writeBuffer(surroundSampleFrame const* _ab, fpp_t const fram
 		{
 			for(ch_cnt_t channel=0; channel<channels(); ++channel)
 			{
-				buf[frame*channels() + channel] = _ab[frame][channel] * master_gain;
+				// Clip the negative side to -0.999999 in order to prevent it from changing sign
```

Comment:
Should this now be `// Clip the negative side to just above -1.0 ...` ?

Reply:
I think so.

Reply:
Fixed!